scraper/visualize.py

```diff
@@ -64,8 +65,8 @@ def compare_products(
 
     product_ids = [product.id for product in products_to_compare]
     product_ids_string = ", ".join(product_ids)
-
-    show_products(products_to_compare, f"Comparing products with ids: {product_ids_string}")
+    
+    show_products(products_to_compare, f"Comparing products with ids: {product_ids_string[:100] + ' ...' if  len(product_ids_string) > 100 else product_ids_string}")
```

Comment:
Addressed really long title of the scatter plot and limited it to first 100 characters. It is mention worthy that title beyond 100 character does not accommodate into screen normally and can not be viewed even since there is no way of adding horizontal scroll in the title of the scatter plot.

Reply:
Can you split this line into multiple lines, so that the ternary operator is not in the f-string?
And after that then please format the code using the settings in file `pyproject.toml`

Reply:
formatted the ternery operator. In fact python-plotly is not able to accommodate long title as shown in `https://stackoverflow.com/questions/78563796/how-to-add-a-scrollbar-to-python-plotly-long-title`.